sdk/core/src/tasks/sboms/github/sync.rs

```diff
@@ -34,20 +63,22 @@ pub struct SyncTask {
 impl TaskProvider for SyncTask {
     /// Builds the Packages Dependencies, Purls, and Unsupported from the GitHub API.
     async fn run(&self, task: &mut Task) -> Result<HashMap<String, String>, Error> {
-        let mut errors = HashMap::new();
-
         let mut repos: Vec<Repo> = match self.github.get_repos().await {
             Ok(value) => value,
             Err(err) => {
-                println!("Error attempting to get the repos: {}", err);
+                println!("Error attempting to get the repos: {:#?}", err);
```

Comment:
Do we want to leave this debug in here on purpose?  My perennial fear around this topic is that we might end up leaking things into logs that we can't predict.

Reply:
No, I can change that back.  Good thinking.